Database_Task.txt

```diff
@@ -1 +1,97 @@
+(1)Creating DATABASE
 
+CREATE DATABASE employee;
```

Comment:
Apply the valid spacing table.
Apply the 3 space for column.
E.g  
CREATE TABLE hobby(
   id INT PRIMARY KEY AUTO_INCREMENT,
   NAME VARCHAR(50)
);